### PR TITLE
Update Kyle Shevlin's instructor landing page

### PR DESCRIPTION
**Pull request by zacjones93**
yeah

Learn web development from Kyle Shevlin on egghead  egghead io
Learn web development from Kyle Shevlin on egghead  egghead io

### CODE REVIEW

File: src/components/card/verticle-resource-card.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const VerticalResourceCard: React.FC<{
             {resource.name}
           </h2>
           <ResourceLink path={resource.path} location={location}>
-            <h3 className="text-lg font-bold leading-tighter py-3">
+            <h3 className="text-lg font-bold leading-tighter py-3 hover:text-blue-700">
```
Comment (zacjones93) [inline src/components/card/verticle-resource-card.tsx:45]: made this consistent with other cards on egghead

File: src/components/search/instructors/kyle-shevlin/index.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                describe
+                location="Kyle Shevlin instructor Landing page"
+              />
+            )
+          })}
+        </div>
+      </section>
+    </div>
+  )
+}
```
Comment (zacjones93) [inline src/components/search/instructors/kyle-shevlin/index.tsx:59]: The magic

File: src/components/search/instructors/kyle-shevlin/index.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+const SearchKyleShevlin = ({instructor}: {instructor: any}) => {
+  const {courses} = instructor
+
+  if (!courses) {
+    return (
+      <div className="max-w-screen-xl mx-auto">
+        <SearchInstructorEssential instructor={instructor} />
+      </div>
+    )
+  }
```
Comment (zacjones93) [inline src/components/search/instructors/kyle-shevlin/index.tsx:21]: returning early with the default layout if courses don't load properly

File: src/components/search/instructors/kyle-shevlin/index.tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            )
+          })}
+        </div>
+      </section>
+    </div>
+  )
+}
+export default SearchKyleShevlin
+
+const CtaCard: React.FC<{location: string; trackTitle: string; resource: any}> =
```
Comment (zacjones93) [inline src/components/search/instructors/kyle-shevlin/index.tsx:62]: copied the CtaCard here because the text needed to be white with this background

**Review by laurosilvacom — APPROVED** [review]
That's such a nice background on the feature course component! ✨

**Pull request merged**